automation/vars/uni05epsilon-adoption.yaml

```diff
@@ -0,0 +1,26 @@
+---
+vas:
+  uni05epsilon-adoption:
+    stages:
+      - path: examples/dt/uni05epsilon/control-plane/networking/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            --timeout=5m
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - path: examples/dt/uni05epsilon/control-plane/networking
```

Comment:
It appears that there is no `examples/dt/uni05epsilon/control-plane/networking` directory and thus no `kustomization.yaml` there to consume.  This is why the CI is failing.